chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/CertifiedProductDownloadableResourceCreatorJob.java

```diff
@@ -194,6 +199,18 @@ private void swapFiles() throws IOException {
         } else {
             LOGGER.warn("Temp XML File was null and could not be moved.");
         }
+
+        if (tempJsonFile != null) {
+            String jsonFilename = getFileName(downloadFolder.getAbsolutePath(),
+                    getFilenameTimestampFormat().format(new Date()), "json");
+            LOGGER.info("Moving " + tempJsonFile.getAbsolutePath() + " to " + tempJsonFile);
+            Path targetFile = Files.move(tempJsonFile.toPath(), Paths.get(jsonFilename), StandardCopyOption.ATOMIC_MOVE);
```

Comment:
Please update the maintenance plan to clear out these files just as we do with the XML files